custom_components/moonraker/__init__.py

```diff
@@ -142,10 +142,23 @@ async def _async_fetch_data(self, query_path, query_object):
         except Exception as exception:
             raise UpdateFailed() from exception
 
+    async def _async_post(self, query_path) -> None:
```

Comment:
I think I would rename it _async_send_data
1. I don't think this is doing a Post. 
1. I think it is a better aligment with _async_fetch_data